system-tests/mono-repo/snapshot

```diff
@@ -0,0 +1 @@
+0
```

Comment:
I found it weird not to have any of this test that assert on having at least an issue. Would it be possible to raise an issue?

Reply:
Yeah I know I tried to induce an issue was harder that what I thought. `eslint-5` seems way more permissive I don't know.